### PR TITLE
Remove state from renderer. Rely on world for state.

### PR DESCRIPTION
**Pull request by tttppp**
This fixes !605 - water displayed as falling even when cells below are full.

Also some wildcard imports, which IntelliJ insists are mandatory (and so
presumably they are desired based on the settings in the .idea directory).

### CODE REVIEW

**Comment by tttppp** [comment]
Ideally we should check the overhead in looking up the region all the time, and perhaps use a cache if it's expensive.

File: rabbit-escape-render/src/rabbitescape/render/WaterRegionRenderer.java [hunk 1]
```diff
@@ -43,21 +29,23 @@
     private final WaterAnimation waterAnimation;
 
     public WaterRegionRenderer(
-        WaterRegion region, 
+        int x,
+        int y,
```
Comment (tttppp) [inline rabbit-escape-render/src/rabbitescape/render/WaterRegionRenderer.java]: This is the bulk of the change in the PR. Replace a copy of the WaterRegion (which sometimes gets outdated and so the graphics go weird) with coordinates so the WaterRegion can be looked up.

**Comment by andybalaam** [comment]
I will fall off my chair if looking up the water region causes a performance problem.  Nice fix!  Fixes #605

**Comment by colonelfazackerley** [comment]
Very nice. a much better idea than how I did it originally. Amazing how I cant see the simpler way until it's shown to me. Thanks @tttppp 

I have to object to the title. The WaterRegionRenderers still hold state (waterHeight, particles). Though this is only state for making things look pretty.

I also now understand the importance of separating formatting and substantive code changes into different diffs @andybalaam

**Pull request merged**